torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2463,6 +2486,36 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # TODO: remove redundant method_tests() entries
                SkipInfo('TestOpInfo', 'test_duplicate_method_tests')),
            sample_inputs_func=sample_inputs_addmm),
+    OpInfo('addmv',
+           dtypes=floating_types(),
+           dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
+           dtypesIfROCM=floating_types_and(torch.half),
+           skips=(
+               # https://github.com/pytorch/pytorch/issues/55539 to track all issues for addmv
+               # failed with a large difference in tensor outputs for below tests
+               SkipInfo('TestCommon', 'test_variant_consistency_eager', dtypes=(torch.float32, torch.complex64)),
```

Comment:
`test_variant_consistency_eager` might pass now, since after setting `supports_inplace_autograd=False`,  it wouldn't test the backward for its inplace variant anymore:

https://github.com/pytorch/pytorch/blob/dd2bccafc599b3931555d95c222809a94d1cbe28/test/test_ops.py#L295-L298

Reply:
Good idea, but I still have those tests failing even with the support_inplace_autograd=False, I'm not totally sure why. For the purpose of porting the op I don't think it's an issue

Reply:
Thank you for your response!
I looked at your latest code, and it seems that the test is only failing on CPU, so my assumption was incorrect.
Sorry, I should've tested locally before commenting.

Reply:
It looks like @mruberry suggestion of changing the sample_inputs generator to check for for_inplace_variant fixed these skips 👍